egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -26,12 +26,16 @@ def GetSumDescriptor(inputs):
     return sum_descriptors
 
 # adds the input nodes and returns the descriptor
-def AddInputLayer(config_lines, feat_dim, splice_indexes=[0], ivector_dim=0):
+def AddInputLayer(config_lines, feat_dim, splice_indexes=[0], ivector_dim=0, idct_mat = None):
```

Comment:
Let's get rid of these changes since they are not necessary for any of your current recipes and are orthogonal to the other changes in this PR, and also may conflict with another PR by Vijay.